cmd/cloud/deployment_test.go

```diff
@@ -862,6 +861,28 @@ deployment:
 		_, err = execDeploymentCmd(cmdArgs...)
 		assert.ErrorContains(t, err, "Invalid --high-availability value")
 	})
+	t.Run("returns an error if cluster-id is provided with implicit standard deployment", func(t *testing.T) {
```

Comment:
Can we add one more test case with type as dedicated if it doesnt exist?

Reply:
nvm actually. the code changes are not impacting that and one of the unit tests already covers the dedicated use case.